xbmc/guilib/imagefactory.cpp

```diff
@@ -46,16 +46,15 @@ IImage* ImageFactory::CreateLoader(const CURL& url)
 
 IImage* ImageFactory::CreateLoaderFromMimeType(const std::string& strMimeType)
 {
-  VECADDONS codecs;
-  ADDON::CBinaryAddonCache &addonCache = CServiceBroker::GetBinaryAddonCache();
-  addonCache.GetAddons(codecs, ADDON::ADDON_IMAGEDECODER);
-  for (auto& codec : codecs)
```

Comment:
Have measured performance (with one addon present).

|  |    |
|--|---|
| With binary addon cache | ~0.000040 sec
| Complete without cache | ~0.005000 sec
| With new on this | ~0.000065 sec